--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -25,7 +25,7 @@ set(TRANSLATION_DIR "${CMAKE_INSTALL_FUL
 add_definitions(-DQT_NO_SIGNALS_SLOTS_KEYWORDS "-DTRANSLATION_DIR=\"${TRANSLATION_DIR}\"" "-DNM_TRAY_VERSION=\"${NM_TRAY_VERSION}\"")
 if (NOT NM_TRAY_XDG_AUTOSTART_DIR)
     #Note: we need the default to be /etc... (not the ${CMAKE_INSTALL_PREFIX}/etc)
-    set(NM_TRAY_XDG_AUTOSTART_DIR "/${CMAKE_INSTALL_SYSCONFDIR}/xdg/autostart")
+    set(NM_TRAY_XDG_AUTOSTART_DIR "${CMAKE_INSTALL_SYSCONFDIR}/xdg/autostart")
 endif ()
 
 message(STATUS "Translations destination dir: ${TRANSLATION_DIR}")
